core/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -194,10 +266,12 @@ public RestStatus status() {
         if (shardFailures.size() == 0) {
             return RestStatus.OK;
         }
-        return RestStatus.status(successfulShards, totalShards, shardFailures.toArray(new ShardOperationFailedException[shardFailures.size()]));
+        return RestStatus.status(successfulShards, totalShards,
+                                 shardFailures.toArray(new ShardOperationFailedException[shardFailures.size()]));
     }
 
     static final class Fields {
+        static final String SNAPSHOT = "snapshot";
```

Comment:
While you are here why not remove the whole `Fields` class and just use strings? That is the "new style". For things that are used in more than one place we'll typically use constants - and if those things are using in parsing the constant will typically be a ParseField, but otherwise we've started to not make these `Fields` classes.